### PR TITLE
Add a few more batch changes templates

### PR DESCRIPTION
**Pull request by malomarrec**
## Test plan

Those are just templates

## App preview:

- [Web](https://sg-web-mm-add-templates.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-aocvaxsoeu.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

**Review by BolajiOlajide — APPROVED** [review]

**Review by eseliger — APPROVED** [review]

**Review by Piszmog — APPROVED** [review]

File: client/web/src/enterprise/batches/batch-spec/edit/library/monorepo-dynamic.batch.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      else
+        echo "I'm running in $PWD" > hello-world.md
+      fi
+    container: alpine:3
+
+changesetTemplate:
+  title: Run in each workspace of a monorepo
+  body: |
+    Run in each workspace of a monorepo
+# Each bracnh needs to be unique, so we're using templating to define them
```
Comment (LawnGnome) [inline client/web/src/enterprise/batches/batch-spec/edit/library/monorepo-dynamic.batch.yaml:28]:
```suggestion
# Each branch needs to be unique, so we're using templating to define them
```

File: client/web/src/enterprise/batches/batch-spec/edit/library/sed.batch.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  # https://docs.sourcegraph.com/batch_changes/references/batch_spec_templating
+  - run: |
+      for file in "${{ join repository.search_result_paths " " }}";
+      do
+        sed -i 's/replace-me/by-me/g;' ${file}
+      done
+    container: alpine:3
+
+changesetTemplate:
+  title: Apply  a regex
```
Comment (LawnGnome) [inline client/web/src/enterprise/batches/batch-spec/edit/library/sed.batch.yaml:18]:
```suggestion
  title: Apply a regex
```

File: client/web/src/enterprise/batches/batch-spec/edit/library/sed.batch.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  body: Apply a regex using sed
+  branch: campaigns/regex
```
Comment (LawnGnome) [inline client/web/src/enterprise/batches/batch-spec/edit/library/sed.batch.yaml:20]: :sob:

```suggestion
  branch: batch-changes/regex
```

**Review by LawnGnome — APPROVED** [review]

File: client/web/src/enterprise/batches/batch-spec/edit/library/sed.batch.yaml (hunk 3 above)
Comment (malomarrec) [reply client/web/src/enterprise/batches/batch-spec/edit/library/sed.batch.yaml]: I'm going with batch-campaigns

**Pull request merged**